docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -0,0 +1,75 @@
+# X-Road Security Server Toolkit configuration guide
+
+**Technical Specification**
+
+Version: 1.0 
+Doc. ID: XRDSST-CONF
+
+| Date       | Version     | Description                                                                  | Author             |
+|------------|-------------|------------------------------------------------------------------------------|--------------------|
+| 10.11.2020 | 1.0         | Initial draft                                                                | Bert Viikmäe       |
+
+
+## Table of Contents
+
+<!-- vim-markdown-toc GFM -->
+* [License](#license)
+* [1 Introduction](#1-introduction)
+* [2 Configuration of X-Road Security Server](#2-configuration-of-x-road-security-server)
+    * [2.1 General](#21-general)
+    * [2.2 Format of configuration file](#22-format-of-configuration-file)
+* [3 Running the X-Road Security Server Toolkit](#3-running-the-x-road-security-server-toolkit)
+    * [3.1 The automatic configuration of a single security server](#31-the-automatic-configuration-of-a-single-security-server)
+
+
+<!-- vim-markdown-toc -->
+
+## License
+
+This document is licensed under the Creative Commons Attribution-ShareAlike 3.0 Unported License. To view a copy of this license, visit http://creativecommons.org/licenses/by-sa/3.0/.
+
+## 1 Introduction
+
+This specification describes automatic configuration of X-Road security servers using X-Road Security Server Toolkit
+
+## 2 Configuration of X-Road Security Server
+
+### 2.1 General
+
+The automatic configuration of X-Road security servers using the X-Road Security Server Toolkit relies on configuration files, where exact configuration values have to be specified
+
+### 2.2 Format of configuration file
+
```

Comment:
This should be inside code block, lines will be merged otherwise.

Reply:
Fixed that one